modules/ckanext-ytp_main/ckanext/ytp/templates/group/snippets/info.html

```diff
@@ -20,7 +20,7 @@ <h1 class="heading">
     {% block description %}
     {% if group.description %}
       <p>
-        {{ h.markdown_extract(group.description, 180) }}
+        {{ h.get_translated(group, 'description')|truncate(180) or h.markdown_extract(group.description, 180)}}
```

Comment:
This propably doesn't render description as markdown?

Reply:
Not sure if it needs to be rendered as markdown in this view, but I'll check if there is get translated version for markdown.